### ISSUE TITLE
Add Content: "PyTorch Fundamentals"

### ISSUE THREAD

**Issue by kRiShNa-429407** [animator/learn-python#643]
Please assign it to me

it includes 

1. Creating various tensors in pytorch
2. operations on tensors

**Comment by akashlogics** [comment]
I would like to. Work on this issue as I'm studying Artificial intelligence and data science and have learned more about Python libraries pyTorch and Tensorflow and many functions. So can you please assign me this problem
and label it GSSoc 24

I hope this will be the first issue I will be solving with full effort as a contributor

@animator Thank you

**Comment by akashlogics** [comment]
@kRiShNa-429407 do you mean we can create our own issue

**Comment by kRiShNa-429407** [comment]
Yes that issue which is not raised by anyone

**Comment by kRiShNa-429407** [comment]
Please assign it my content is ready

**Comment by kRiShNa-429407** [comment]
Please assign it to me
